modules/tensor_mechanics/src/materials/RecomputeRadialReturn.C

```diff
@@ -19,24 +19,24 @@ InputParameters validParams<RecomputeRadialReturnStressIncrement>()
   return params;
 }
 
-RecomputeRadialReturnStressIncrement::RecomputeRadialReturnStressIncrement(const InputParameters & parameters) :
-    RecomputeGeneralReturnStressIncrement(parameters),
-    _output_iteration_info(getParam<bool>("output_iteration_info")),
-    _output_iteration_info_on_error(getParam<bool>("output_iteration_info_on_error")),
-    _relative_tolerance(parameters.get<Real>("relative_tolerance")),
-    _absolute_tolerance(parameters.get<Real>("absolute_tolerance"))
+RecomputeRadialReturn::RecomputeRadialReturn(const InputParameters & parameters) :
+  RecomputeGeneralReturn(parameters),
+  _output_iteration_info(getParam<bool>("output_iteration_info")),
```

Comment:
OK.  I'll change it to four spaces.  We all realize this is silly, right?  Why are four spaces in an initialization list better than two (used everywhere else)?